tools/gmhazard_scripts/db_creation/empirical_db/calc_distances_flt.py

```diff
@@ -153,6 +153,12 @@ def load_args():
         help="List of stations for a specific domain. Source to site distances "
         "will be calculated for all stations in the station_file.",
     )
+    parser.add_argument(
+        "--directivity",
+        type=lambda x: (str(x).lower() == 'true'),
```

Comment:
store_true is the option you are looking for.

Reply:
Well, or store_false

Reply:
I thought that `store_false` was what I used but apparently, Joel one also seems to work as expected

https://stackoverflow.com/a/46951029 :exploding_head: